cityofaustin/atd-moped
# Add foreign key relationships to moped_proj_phases

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9242

Similar to moped_proj_milestones: https://github.com/cityofaustin/atd-moped/pull/654

I was unable to drop the phase_name column completely, because the project_list_view depends on it. I opened a new issue to address that tech debt: https://github.com/cityofaustin/atd-data-tech/issues/9350

I updated the seed data to have a phase_id in our seed projects. 

## Testing
**URL to test:**  test locally or moped test

**Steps to test:**
1. Add new phase in milestone table
2. Design and Construction have subphases, try adding one too. 
3. experiment with setting a current phase. Toggle another phase as current phase, only one phase can be current at any time. (But there can also be no current phases)
4. If the phase you set also matches one of our statuses, you should see the badge update to match the phase name
5. If you remove all current phases, you will see the badge label switch to active. 



---
#### Ship list
